src/app/trade-creator-page/trade-creator-table/trade-creator-table.component.html

```diff
@@ -96,25 +96,36 @@
   <!-- Founder's Reward Warning -->
   <div
     *ngIf="creatorCoinTrade.getFoundersRewardPercent() > 0 && !creatorCoinTrade.isBuyingOwnCoin()"
-    class="py-3 border-bottom text-muted d-flex"
+    class="py-3 text-muted d-flex"
   >
     <div class="mr-10px">
-      <i class="fas fa-heart fa-md text-grey9"></i>
+      <i class="fas fa-heart fa-md text-grey5"></i>
     </div>
-    <div [ngClass]="{ 'text-danger': creatorCoinTrade.showFounderRewardWarning() }">
+    <div [ngClass]="{ 'text-danger': creatorCoinTrade.showFounderRewardWarning() }" class="text-grey5">
       @{{ creatorCoinTrade.creatorProfile.Username }} will receive
       <span class="font-weight-semibold">{{ creatorCoinTrade.getFoundersRewardPercent() | number: "0.0-9" }}%</span>
       of your purchase as a
       <span style="white-space: nowrap">
         founder reward
         <i
           (click)="tooltip.toggle()"
-          class="fas fa-info-circle text-greyC fs-15px global__tooltip-icon"
+          class="fas fa-info-circle text-grey5 fs-15px global__tooltip-icon"
           matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
           [matTooltip]="TradeCreatorTableComponent.FOUNDER_REWARD_EXPLANATION"
           #tooltip="matTooltip"
         ></i>
       </span>
     </div>
   </div>
+  <div *ngIf="!userFollowingCreator" class="py-1 border-bottom text-muted d-flex text-grey9 follow-on-buy">
```

Comment:
add to the *ngIf something to check that  the loggedInUser isn't purchasing themselves.